src/proxy/query.go

```diff
@@ -145,7 +145,7 @@ func (spanner *Spanner) ComQuery(session *driver.Session, query string, bindVari
 				log.Error("proxy.show.engines[%s].from.session[%v].error:%+v", query, session.ID(), err)
 				status = 1
 			}
-		case sqlparser.ShowTablesStr, sqlparser.ShowFullTablesStr:
+		case sqlparser.ShowTablesStr:
 			// Support for SHOW FULL TBALES which can be parsed used by Navicat
 			// TODO: need to support: SHOW [FULL] TABLES [FROM db_name] [like_or_where]
```

Comment:
The TODO comment should be removed and we can add a comment `support: SHOW [FULL] TABLES [FROM db_name] [like_or_where]`.
Show [full] columns ... should also add a comment.

Reply:
At present, we only support the syntax of `show full tables`, but actually `show full tables` is treated as `show tables`.

Reply:
> The TODO comment should be removed and we can add a comment `support: SHOW [FULL] TABLES [FROM db_name] [like_or_where]`.
> Show [full] columns ... should also add a comment.

At present, we only support the syntax of show full tables, but actually show full tables is treated as show tables.

Reply:
Got.